Sofa/framework/Core/src/sofa/core/visual/DisplayFlags.cpp

```diff
@@ -96,6 +97,25 @@ std::ostream& FlagTreeItem::write(std::ostream &os) const
     return os;
 }
 
+void FlagTreeItem::showUnknownTokenMessage(const std::map<std::string, bool, FlagTreeItem::ci_comparison>& parseMap, std::string token) const
+{
+    sofa::type::vector<std::string> allFlagNames;
+    allFlagNames.reserve(parseMap.size());
+    for (const auto& [flagName, _] : parseMap)
+    {
+        allFlagNames.emplace_back(flagName);
+    }
+
+    std::stringstream tmp;
+    tmp << "Unknown token '" << token << "'" << ". The closest existing flags are:" << msgendl;
```

Comment:
"token" sound like programmer's slang. 
```suggestion
    tmp << "Unknown flag '" << token << "'" << ". The closest existing ones:" << msgendl;
```

Reply:
I agree, I copied/pasted this one.